app/javascript/components/JobEdit.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+class JobEdit extends React.Component {
+  render() {
+    var errors = null;
+      if(this.props.errors.length > 0) {
+        var errorsList = [];
+        for(var i = 0; i < this.props.errors.length; ++i) {
+          errorsList.push(<li key={"error-" + i}>{this.props.errors[i]}</li>)
+        }
```

Comment:
Neat error handling; but in general, we should use a functional approach (especially for consistency). Replace this with a `map`.

Reply:
Ok